app/graphql/mutations/rock_and_pebbles/discontinue_rock_pebble_relationship.rb

```diff
@@ -0,0 +1,33 @@
+module Mutations
+    module RockAndPebbles
+      class DiscontinueRockPebbleRelationship < ::Mutations::BaseMutation
+        argument :id, ID, required: true
+        argument :reason, String, required: true
+        argument :userId, ID, required: true
+        type Types::RockAndPebbleType
+
+        def resolve(attributes)
+          rock_and_pebble = RockAndPebble.find(attributes[:id])
+          send_pebble_email(rock_and_pebble, attributes)
+
+          rock_and_pebble.update(active: false)
+          rock_and_pebble
+        end
+
+        private
+
+        def send_pebble_email(rock_and_pebble, attributes)
+          user = User.find(attributes[:user_id])
+          if user.id == rock_and_pebble.rock_id
+            reason = attributes[:reason]
+            info = rock_and_pebble.rock_pebble_info
```

Comment:
Another small optimization: because this is the same in both the `if` and `else`, this could be moved up to the top of the method and lines 26-27 could be deleted

Reply:
Haha I thought of this after pushing and was being lazy! Changes pushed up